src/cloud_provider/digitalocean/kubernetes/mod.rs

```diff
@@ -1100,6 +1099,16 @@ impl<'a> DOKS<'a> {
             }
         }
 
+        info!("Empty Qovery managed object storage buckets");
+        if let Err(e) = self.spaces.empty_bucket(self.kubeconfig_bucket_name().as_str()) {
```

Comment:
I guess you can also use the `delete` bucket function, bqased on spaces stratgey, which should be `empty`, deleting the bucket will lead to only empty it => https://github.com/Qovery/engine/blob/729e96c69ea6a8a6f1426bb0bc201e5bed338b83/src/object_storage/spaces.rs#L275

Reply:
Since it's DO bucket with the deletion issue and local tests always use the same name, it's mandatory to only empty it. Pleco will delete empty buckets anyway is not used for a long time.